mps-sync-plugin-lib/build.gradle.kts

```diff
@@ -14,6 +14,8 @@ val mpsZip by configurations.creating
 dependencies {
     implementation(kotlin("stdlib-jdk8"))
     implementation(libs.kotlin.logging.microutils)
+    implementation(libs.kotlin.serialization.json)
+    implementation(libs.jackson.dataformat.xml)
```

Comment:
Same as https://github.com/modelix/modelix.mps-plugins/pull/94/files#r1637822282

Reply:
fixed in 68fa9ad499e56e66d84e58ab26c29184112b4241